packages/kbn-securitysolution-autocomplete/src/check_empty_value/index.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import * as i18n from '../translations';
+
+// TODO: I have to use any here for now, but once this is available below, we should use the correct types, https://github.com/elastic/kibana/issues/105731
+// import { IFieldType } from '../../../../../../../src/plugins/data/common';
+type IFieldType = any;
```

Comment:
Nit: https://github.com/elastic/kibana/pull/103530 is merged

Reply:
Thanks, for letting me know that merged. They created a Kibana ticket for me I assigned it to myself. I have to fix these across all the packages and that will be a follow up all at once for these which I included in the comments of this PR:
https://github.com/elastic/kibana/issues/105731